File: 0001-two-sum/solution.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        #    seen={}

        #    for i,num in enumerate(nums):

        #        c=target-num

        #        if c in seen:
        #            return [seen[c],i]
                
        #        seen[num]=i

        #n=len(nums)

        #for i in range(n):
        #    for j in range(i+1,n):
        #        if nums[i]+nums[j]==target:
        #           return [i,j]
                    
        seen={}

        for i,num in enumerate(nums):
            c=target-num

            if c in seen:
                return [seen[c],i]
            seen[num]=i
        
            
        

File: 0001-two-sum/solution.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        for(int idx=0;idx<nums.length;idx++){
            for(int taridx=idx+1;taridx<nums.length;taridx++){
                if(nums[idx]+nums[taridx]==target){
                    return new int[]{idx,taridx};
                }
            }
        }
        return new int[]{};
    }
}
